Resources/Prototypes/Entities/Objects/Tools/jetpacks.yml

```diff
@@ -0,0 +1,311 @@
+- type: entity
+  parent: GasTankBase
+  abstract: true
+  id: BaseJetpack
+  name: Jetpack
+  description: It's a jetpack.
+  components:
+    - type: Sprite
+      sprite: Objects/Tanks/Jetpacks/blue.rsi
+      state: icon
+      netsync: false
+    - type: UserInterface
+      interfaces:
+        - key: enum.SharedGasTankUiKey.Key
+          type: GasTankBoundUserInterface
+    - type: Clothing
```

Comment:
can you add a size to this so its heavier than 15 weight. i dont think players should be running around with these in their bags without sacrificing a lot of weight

Reply:
Could probably make it like 30 or 50. Generally they get used as backpacks anyway.

Reply:
Alright made it 100 so they don't fit in bags normally, except for captain's and mini.

Reply:
Captains jetpack should be substantially smaller in size than others as it is intended to fit into a backpack